### ISSUE TITLE
Terraform 0.15.0 panic

### ISSUE THREAD

**Issue by sigurdhj** [wgebis/terraform-provider-mailgun#11]
### Terraform Version
Terraform v0.15.0
on linux_amd64
+ provider registry.terraform.io/dnsimple/dnsimple v0.5.1
+ provider registry.terraform.io/hashicorp/azuread v1.1.1
+ provider registry.terraform.io/hashicorp/azurerm v2.56.0
+ provider registry.terraform.io/hashicorp/null v3.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/terraform-providers/dnsimple v0.4.0
+ provider registry.terraform.io/terraform-providers/mailgun v0.4.3
+ provider registry.terraform.io/wgebis/mailgun v0.5.6

### Affected Resource(s)
- mailgun_route
- mailgun_domain

### Terraform Configuration Files
```hcl
resource "mailgun_domain" "this" {
  name   = "${local.customer_name}.${local.domain}"
  region = "eu"

  lifecycle {
    ignore_changes = [smtp_password, spam_action]
  }
}

resource "mailgun_route" "this" {
  region      = "eu"
  priority    = "0"
  description = "${local.customer_name} mail"
  expression  = "match_recipient('(.*)@${local.customer_name}.${local.domain}')"
  actions = [
    "store(notify='https://${local.customer_name}.${local.domain}/agent/api/webhook/mailgun/${random_password.mailgun_token.result}')",
    "stop()"
  ]
}
```
### Panic Output
https://gist.github.com/sigurdhj/29f38e59415acfe888e0204e3b1f5c23

### Expected Behavior
Should create domain and route in mailgun

### Actual Behavior
Did not create domain and route in mailgun

**Comment by wgebis** [comment]
@sigurdhj looks that it's because latest version of Mailgun plugin uses to old Terraform SDK, it will me fixed in next version of plugin.

**Comment by wgebis** [comment]
It refers to the old version of Terraform SDK, here is update: #15.

**Comment by wgebis** [comment]
Fixed in v0.6.0
